### PR TITLE
add option for alternate compression of artifact (xz)

### PR DESCRIPTION
**Pull request by rubyisbeautiful**

### CODE REVIEW

**Comment by rubyisbeautiful** [comment]
Hey @ctaintor I didn't add any tests because of your comment in there.  Let me know if you want one/some.  Thanks for making this, was very helpful to us.

File: lib/capistrano-rails-artifact/version.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
 module CapistranoRailsArtifact
-  VERSION = '1.0.0'
+  VERSION = '1.0.1'
```
Comment (ctaintor) [inline lib/capistrano-rails-artifact/version.rb]: can you make this 1.1.0? it's adding a new feature :)

File: capistrano-rails-artifact.gemspec [hunk 2]
```diff
@@ -8,8 +8,8 @@ Gem::Specification.new do |spec|
   spec.version       = CapistranoRailsArtifact::VERSION
   spec.authors       = ["Case Taintor"]
   spec.email         = ["case.taintor@klarna.com"]
-  spec.summary       = %q{A gem to allow you to deploy using a .tar.gz}
-  spec.description   = %q{A gem to allow you to deploy using a .tar.gz}
+  spec.summary       = %q{A gem to allow you to deploy using a .tar.gz or .xz}
+  spec.description   = %q{A gem to allow you to deploy using a .tar.gz or .xz}
```
Comment (ctaintor) [inline capistrano-rails-artifact.gemspec]: .tar.xz?

**Comment by ctaintor** [comment]
Thanks for the PR!

**Comment by rubyisbeautiful** [comment]
Done and thanks for those good catches - I was in a big hurry, your gem helped us (Tripcase) out in pinch.

**Comment by ctaintor** [comment]
Thanks! Tagged the new version and pushed to [rubygems](https://rubygems.org/gems/capistrano-rails-artifact)

**Pull request merged**